src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -1890,14 +1890,14 @@ CmdSketcherConstrainCoincident::CmdSketcherConstrainCoincident()
     sAppModule      = "Sketcher";
     sGroup          = "Sketcher";
     sMenuText       = QT_TR_NOOP("Constrain coincident");
-    sToolTipText    = QT_TR_NOOP("Create a coincident constraint on the selected item");
+    sToolTipText    = QT_TR_NOOP("Create a coincident constraint between points. If instead circles, arcs, ellipses or arcs of ellipse are selected, it will create a concentric constraint.");
```

Comment:
I'd go with something like:
```suggestion
    sToolTipText    = QT_TR_NOOP("Create a coincident constraint between points, or a concentric constraint between circles, arcs, and ellipses");
```